owncloud/owncloud.py

```diff
@@ -804,6 +1022,105 @@ def delete_attribute(self, app, key):
             return True
         raise ResponseError(res)
 
+
+    def get_apps(self):
+    	""" List all enabled apps through the provisioning api.
+	    If this fails, try to run enable_app_ajax('provisioning_api') first.
+	    Returns a dict of apps, with values True/False, representing the enabled state.
+	"""
+	ena_apps = {}
```

Comment:
This project isn't ready for Py3.
So far I used 4 spaces for indentation all over the project (https://github.com/owncloud/pyocclient/blob/master/owncloud/owncloud.py#L3). I also had the default settings for pylint, not sure which they are (whether PEP8 or something else).